.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,17 @@
+name: Node.js CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install yarn
```

Comment:
This is not required, `yarn` is preinstalled in the environment.

Reply:
I saw no mention of it anywhere so assumed it wasn't.